SUMMARY.md

```diff
@@ -41,34 +41,35 @@
-          * [Modal component](products/editions/enterprise-edition/b2b-suite/guides/storefront/modal-component.md)
-          * [Product Search](products/editions/enterprise-edition/b2b-suite/guides/storefront/product-search.md)
+         * [Installation](products/editions/enterprise-edition/b2b-suite/guides/installation/README.md)
+         * [Core](products/editions/enterprise-edition/b2b-suite/guides/core/README.md)
+           * [Dependency injection](products/editions/enterprise-edition/b2b-suite/guides/core/dependency-injection.md)
+           * [Store API](products/editions/enterprise-edition/b2b-suite/guides/core/store-api.md)
+           * [REST API](products/editions/enterprise-edition/b2b-suite/guides/core/rest-api.md)
+           * [Listing service](products/editions/enterprise-edition/b2b-suite/guides/core/listing-service.md)
+           * [CRUD service](products/editions/enterprise-edition/b2b-suite/guides/core/crud-service.md)
+           * [Assignment service](products/editions/enterprise-edition/b2b-suite/guides/core/assignment-service.md)
+           * [Audit Log](products/editions/enterprise-edition/b2b-suite/guides/core/audit-log.md)
+           * [Exception](products/editions/enterprise-edition/b2b-suite/guides/core/exception.md)
+           * [Currency](products/editions/enterprise-edition/b2b-suite/guides/core/currency.md)
+           * [Entity based ACL](products/editions/enterprise-edition/b2b-suite/guides/core/entity-acl.md)
+           * [Line Item List](products/editions/enterprise-edition/b2b-suite/concepts/line-item-list.md)
+           * [Overloading classes](products/editions/enterprise-edition/b2b-suite/guides/core/overload-classes.md)
+           * [Store Front Authentication](products/editions/enterprise-edition/b2b-suite/guides/core/authentication.md)
+         * [Storefront](products/editions/enterprise-edition/b2b-suite/guides/storefront/README.md)
+           * [Ajax Panel](products/editions/enterprise-edition/b2b-suite/guides/storefront/ajax-panel.md)
+           * [Product Search](products/editions/enterprise-edition/b2b-suite/guides/storefront/product-search.md)
+           * [Complex views](products/editions/enterprise-edition/b2b-suite/guides/storefront/complex-views.md)
+           * [Modal component](products/editions/enterprise-edition/b2b-suite/guides/storefront/modal-component.md) 
+           * [Company](products/editions/enterprise-edition/b2b-suite/guides/storefront/company.md)
+           * [Acl Routing](products/editions/enterprise-edition/b2b-suite/guides/storefront/acl-routing.md)
+           * [Extending the storefront](products/editions/enterprise-edition/b2b-suite/guides/storefront/how-to-extend-the-storefront.md)
+         * [Administration](products/editions/enterprise-edition/b2b-suite/guides/administration/README.md) 
+         * [Example Plugins](products/editions/enterprise-edition/b2b-suite/guides/example-plugins/README.md)
```

Comment:
please sort it alphabetically.

Reply:
the ticket specify to keep the provided structure in the epic